_tags

```diff
@@ -4,8 +4,9 @@ true: warn(@5@8@10@11@12@14@23-24@26@29@40), bin_annot, safe_string, debug
 <node_modules/**>: -traverse
 "formatTest": -traverse
 "src": include
-<src/*>: package(menhirLib unix compiler-libs.common ocamlbuild findlib merlin_extend)
+<src/*>: package(menhirLib unix compiler-libs.common ocamlbuild findlib merlin_extend ppx_tools.metaquot result dynlink findlib.dynload)
```

Comment:
ppx_tools.metaquot does help a lot with keeping the code concise and compatible across OCaml AST changes.  That said, once the ocaml-migrate-parsetree work is merged that library can be used to smooth over AST side.